packages/siop-oid4vp/lib/helpers/SIOPSpecVersion.ts

```diff
@@ -43,7 +44,7 @@ export const authorizationRequestVersionDiscovery = (authorizationRequest: Autho
       !authorizationRequestCopy.registration_uri &&
       !authorizationRequestCopy.registration &&
       !(authorizationRequestCopy.claims && 'vp_token' in authorizationRequestCopy.claims) &&
-      authorizationRequestCopy.response_mode !== ResponseMode.POST // Post has been replaced by direct post
+      authorizationRequestCopy.response_mode !== ResponseMode.DIRECT_POST // Post has been replaced by direct post
```

Comment:
Why is this being changed? it is incorrect to be clear. This is a v13 validation, that had POST not direct_post